coffee-bom/pom.xml

```diff
@@ -42,6 +42,7 @@
 		<version.microprofile-reactive-messaging-api>2.0.1</version.microprofile-reactive-messaging-api>
 		<version.microprofile-reactive-streams-operators-api>2.0</version.microprofile-reactive-streams-operators-api>
 		<version.org.eclipse.microprofile.metrics>3.0</version.org.eclipse.microprofile.metrics>
+		<version.org.eclipse.microprofile.health>4.0</version.org.eclipse.microprofile.health>
```

Comment:
This release aligns with Jakarta EE 9.1, so it won’t work with earlier versions of Jakarta or Java EE.

https://github.com/eclipse/microprofile-health/blob/main/spec/src/main/asciidoc/release_notes.asciidoc#incompatible-changes

Reply:
fixed to 3.1